src/backend/index.js

```diff
@@ -0,0 +1,6 @@
+const http = require('http');
```

Comment:
I thought the backend is a hapi js web server. You don't have to create files if you don't need it. Remember, that is why we have other PRs